glacierscript.py

```diff
@@ -816,11 +886,25 @@ def withdraw_interactive():
     #### Calculate Transaction ####
     print "\nCalculating transaction...\n"
 
-    unsigned_tx = create_unsigned_transaction(
-        source_address, addresses, redeem_script, txs)
+    if RE_SIGN_MODE is not 1:
```

Comment:
I'm no Python expert but your use of Booleans like this doesn't seem like best practice. Wouldn't it be better to set the variable to True or False and test using `if RE_SIGN_MODE:` and `if not RE_SIGN_MODE:`

Reply:
I'm no Python expert either...I initially set out to use True/False, but after discovering that in Python 2 (which glacier is written for) these booleans are mutable I switched to 0 and 1 (though there should be low probability of mutability in practice). https://stackoverflow.com/questions/2764017/is-false-0-and-true-1-in-python-an-implementation-detail-or-is-it-guarante

I don't know what best practice is...if it is using True/False I can obviously clean things up as booleans for consolidation/readability.

Reply:
That is pretty crazy of Python 2 but as we're not going to be reassigning True/False to anything other than 1/0 respectively, I see no issue with using them, and it makes it clearer that the variable is a boolean and not an integer.

Even if using 0/1, testing the value can still use `if RE_SIGN_MODE:` without the `is 1` noise.